google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -66,16 +70,55 @@ public static Builder newBuilder() {
     return new Builder();
   }
 
-  /** Create a new builder preconfigured to connect to the Bigtable emulator. */
+  /**
+   * Create a new builder preconfigured to connect to Bigtable emulator when hostname and port
+   * number are provided through BIGTABLE_EMULATOR_HOST environment variable.
+   */
+  public static Builder newBuilderForEmulator() {
```

Comment:
@igorbernstein2 
Q: What are your thoughts on making user set VM args, to be able to use `Builder#newBuilderForEmulator()`?

Q: Should admin & data client can share utility class? because till this point, it's logically separate clients within one module.

Reply:
Q1. We should keep it as an environment variable. The main reason we are adding this feature is to be consistent with the emulator docs.

Q2. Let's keep it separate for now and discuss what other things need to be merge